NostalgiaForInfinityNext.py

```diff
@@ -2423,14 +2423,41 @@ def load_hold_trades_config(self):
                 log.error("Failed to load JSON from %s: %s", hold_trades_config_file, exc)
             else:
                 trade_ids = hold_trades_config.get("trade_ids")
+                trade_pairs = hold_trades_config.get("trade_pairs")
 
             if not trade_ids:
                 return
 
             open_trades = {
                 trade.id: trade for trade in Trade.get_trades_proxy(is_open=True)
             }
-
+            if isinstance(trade_pairs, dict):
+                # New syntax
+                for trade_pair, profit_ratio in trade_pairs.items():
+                    try:
+                        trade_pair = str(trade_pair)
```

Comment:
Casting anything to a string, strings, ints, floats, dictionaries, will always returrn strings, it will never raise ``TypeError``.

Maybe check for `trade_pair.endswith(f"/{the_bot_currency_config}")`
Please find the correct config variable name, I can't recall which is it.

Reply:
so basically `trade_pair.endswith(f"/{self.stake_currency}")`

Reply:
Done for unecessary string type 
Added check stake_currency fit in end of trade_pairs